js/station/index.html

```diff
@@ -0,0 +1,9 @@
+
+<div data-ng-repeat="song in SongIndexViewModel.songs">
+  <!-- the idshow is define in app.js that links to /songs/:id url to show that specific song-->
+  <p><a data-ui-sref="idShow({id: song.id})">{{song.title}}</a></p>
+  <hr>
+</div>
+<!-- <div data-ni-if="songs">
```

Comment:
lots of commented out / dead code in your app, always try to keep that to a minimum.